### PR TITLE
BitPay invoice

### PR DESCRIPTION
**Pull request by nao20010128nao**
Supports BitPay invoice and coding errors.   
Koto OW is removed.

### CODE REVIEW

File: cordovaProj/package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "description": "The easiest cryptocurrency wallet",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "android-init": "cordova platfom add android",
```
Comment (MissMonacoin) [inline cordovaProj/package.json]: typo

File: js/extension.js [hunk 2]
```diff
@@ -16,7 +16,7 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 const storage = require("./storage.js")
-const hdkey = require('ethereumjs-wallet/hdkey')
+const hdkey = require('ethereumjs-wallet/dist/hdkey')
```
Comment (MissMonacoin) [inline js/extension.js]: Why is this change needed?

Comment (nao20010128nao) [reply js/extension.js]: Required to start debug. It broke gulp and prevents starting

Comment (MissMonacoin) [reply js/extension.js]: All minification is done by webpack now so I think it's better to import unminified(w/o dist) version.

Comment (nao20010128nao) [reply js/extension.js]: Reverted.
